__tests__/app.test.js

```diff
@@ -111,4 +111,51 @@ describe('GET /api/articles', () => {
             expect(body.msg).toBe('Route not found')
         })
     })
-})
\ No newline at end of file
+})
+describe('GET /api/articles/:article_id/comments', () => {
+    test('200: responds with an empty array if the article exists, but there are no comments for it', () => {
+    return request(app)
+    .get('/api/articles/2/comments')
+    .expect(200)
+    .then(({body}) => {
+        const { comments } = body
+        expect(comments).toEqual([])
+    })
+    })
+    test('200: responds with an array of comments for the given article_id of which each comment should have the relevant properties', () => {
+        return request(app)
+        .get('/api/articles/3/comments')
+        .expect(200)
+        .then(({body}) => {
```

Comment:
We should always check the length of the array when asserting inside a for each because if an array is empty it will be a false positive 🙂